packages/core/src/create/release.ts

```diff
@@ -132,6 +120,7 @@ export const createReleaseJson = async (
   };
 
   for (const [locale, strings] of Object.entries(releaseDetails.catalog)) {
+    // @ts-expect-error
```

Comment:
can you comment what error is being suppressed here and why?

Reply:
Interesting, it would appear it has to do with the files & media Metaplex overrides you were wondering about below...